.github/workflows/release.yml

```
@@ -600,6 +600,18 @@ jobs:
             build/SHASUMS256.txt
             build/SHASUMS512.txt
 
+      - name: Trigger Homebrew Formula Update
+        if: needs.check-releases.outputs.cli-should-release == 'true' && needs.release-cli.result == 'success'
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2002:style:1:14: Useless cat. Consider 'cmd < file | ..' or 'cmd file | ..' instead

(shellcheck)